### PR TITLE
Add first pass at json schema for set details

### PR DESCRIPTION
**Pull request by justinkchen**
From: https://docs.google.com/document/d/1QoDutLlE4fv8iAubnp3xxi-U9lAhQ4UiEeiodPROYTQ/edit#

Validated through here: 
https://ui-schema.bemit.codes/examples

Following documentation from here: https://json-schema.org/specification-links.html#draft-7

Allow for Tokensets users to customize the following:
Set Icon
Manager Icon
Manager Name
Set Description
Manager Description

@asoong

### CODE REVIEW

File: set-details.schema.json [hunk 1]
```diff
@@ -0,0 +1,185 @@
+// 20210310004324
+// https://github.com/SetProtocol/uniswap-tokenlist/blob/main/set-details.schema.json
+// You can paste the below JSON into a schema validator such as https://www.jsonschemavalidator.net/ to
+// see if your JSON data is following the correct format
+// For example:
+// {
+//   name: "My Set Details",
+//     timestamp: "2012-04-23T18:25:43.511Z",
+//     version: {
```
Comment (controtie) [inline set-details.schema.json:9]: this version is for the individual set? or for the entire set list?

File: set-details.schema.json [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+//       "patch": 0
+//     },
+//     setDetails: {
+//       managerName: "Crypto Manager",
+//       managerDescription: "This is my description",
+//       setDescription: "This is the description of my Set",
+//       setIcon:  "ipfs://QmXfzKRvjZz3u5JRgC4v5mGVbm9ahrUiB4DgzHBsnWbTMM",
+//       managerIcon:  "ipfs://QmXfzKRvjZz3u5JRgC4v5mGVbm9ahrUiB4DgzHBsnWbTMM",
+//     },
+//     signature: "0x21fbf0696d5e0aa2ef41a2b4ffb623bcaf070461d61cf7251c74161f82fec3a4370854bc0a34b3ab487c1bc021cd318c734c51ae29374f2beb0e6f2dd49b4bf41c"
```
Comment (controtie) [inline set-details.schema.json:21]: what's the intended use of this signature? checking if the manager address matches the signature address?

File: set-details/0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "DeFi Pulse Index",
+    "setSymbol": "DPI",
+    "setDecimals": 18,
+    "setIcon": "https://raw.githubusercontent.com/SetProtocol/uniswap-tokenlist/main/assets/dpi.svg",
+    "setDescription": "Pulse Inc.",
+    "managerName": "Pulse Inc.",
```
Comment (asoong) [inline set-details/0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json:9]: DeFi Pulse?

File: set-details/0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "DeFi Pulse Index",
+    "setSymbol": "DPI",
+    "setDecimals": 18,
+    "setIcon": "https://raw.githubusercontent.com/SetProtocol/uniswap-tokenlist/main/assets/dpi.svg",
+    "setDescription": "Pulse Inc.",
```
Comment (asoong) [inline set-details/0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json:8]: Add the current description? As much as we can so the the featured set and custom set are going to be the same

**Review by asoong — APPROVED** [review]

**Pull request merged**